Tools/scripts/deepfreeze.py

```diff
@@ -170,6 +170,8 @@ def generate_bytes(self, name: str, b: bytes) -> str:
     def generate_unicode(self, name: str, s: str) -> str:
         if s in identifiers:
             return f"&_Py_ID({s})"
+        if re.match(r'^[A-Za-z0-9_]+$', s):
```

Comment:
Unfortunately this also matches an identifier followed by a `\n` (newline).

Reply:
Fixed now, I need to learn regex better :)